### DISCUSSION TITLE
Class names in n:class starting with a hyphen are compiled incorrectly

### DISCUSSION THREAD

**Discussion by zraly** [nette/latte#371] [General]
Version: 3.0.18

### Bug Description

`n:class="$foo ? 'font-bold', -top-0.5"` is incorrectly compiled to `[$foo ? 'font-bold' : null, -'top-0.5']`.
The problem is if the class name starts with a hyphen.

The error it produces is `Unsupported operand types: string * int`

### Expected behavior

`n:class="$foo ? 'font-bold', -top-0.5"` is compiled to `[$foo ? 'font-bold' : null, '-top-0.5']`

**Comment by dg** [comment]
You have to put such identifiers in quotes 
https://latte.nette.org/en/syntax#toc-strings-without-quotation-marks
